plugins.d/python.d.plugin

```diff
@@ -515,6 +519,9 @@ def parse_cmdline(directory, *commands):
             mods.append(cmd.replace(".chart.py", ""))
         else:
             try:
+                # FIXME for some reason this overwrites the module configuration
```

Comment:
But this is only for debugging when invoking `python.d.plugin debug 1`. If you don't pass `debug some_number` everything works as expected.